### PR TITLE
Get login duration from backend

### PR DESCRIPTION
**Pull request by ErinClaudio**
### What github issue is this PR for, if any?
Resolves #https://github.com/rubyforgood/casa/issues/4725

### What changed, and why?

- the `config.timeout_in = 3.hours ` has been passed into [here](https://github.com/rubyforgood/casa/pull/4693/files#diff-c51b651f39074e2d51508b09f96b6b3d8f49f719589fd6b0a5b6d72d9ec324b1)
- This hardcoded value is now accessible as an instance variable 

### How will this affect user permissions?
- Volunteer permissions: login duration 
- Supervisor permissions: login duration 
- Admin permissions: login duration 

### How is this tested? (please write tests!) 💖💪
[here](https://github.com/rubyforgood/casa/blob/ea0fe12922e1b1662b3c00d686bc29b2836f19cb/spec/system/application/timeout_warning_spec.rb)

I will need to include the Jasmine gem per Shen's guidance to create a test that simulates the passage of time. 
This task has been created as a new issue ticket.
This is why the second test has not yet been completed.

### Screenshots please :)
Screenshot 2023-08-03 at 11 14 48 AM

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

Is it better to list more or fewer reviewers? Not sure what is best practice for Casa 
Thank you

### CODE REVIEW

**Comment by ErinClaudio** [comment]
- write system tests

File: spec/system/application/timeout_warning_spec.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+      sign_in user
+    end
+    it "renders the seconds before logout as a javascript variable" do
+      visit "/"
+      parsed_page = Nokogiri::HTML(page.html)
+      expect(parsed_page.at("script").text.strip).to include(user.timeout_in.in_seconds.to_s)
+    end
+
+    xit "warns the user two mintues before logout", js: true do
+    end
```
Comment (littleforest) [inline spec/system/application/timeout_warning_spec.rb:16]: @ErinClaudio this looks good! Can you just remove this spec on line 15 if we are not going to test for it?

Comment (ErinClaudio) [reply spec/system/application/timeout_warning_spec.rb:16]: you got it and Thank you

Comment (ErinClaudio) [reply spec/system/application/timeout_warning_spec.rb]: @littleforest Thank you for the review. When you have a moment please

**Pull request merged**